src/components/progress-flow/progress-flow-item/progress-flow-item.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+    Component,
+    Element,
+    Event,
+    EventEmitter,
+    h,
+    Prop,
+} from '@stencil/core';
+import { FlowItem } from '../progress-flow.types';
+
+/**
+ * @private
+ */
+@Component({
+    tag: 'limel-progress-flow-item',
+    shadow: false,
```

Comment:
What is the reason for having this set to `false`? :slightly_smiling_face:

Reply:
We didn't want to have the styles such as `active` and so forth, inside the child component. This made it easier to handle those UI design things in the same place, in the parent component. There was also a CSS issue regarding the order of steps. If shadow was true, then we had to use `reverse` in our flex container, which made a stupid tabbing effect. Also created some z-index rendering for the pseudo elements that create the arrows. So if this is not a critical issue, please don't change it.